js/app.js

```diff
@@ -0,0 +1,180 @@
+// 1
+function setAlarm(employed, vacation) {
+    return employed && !vacation;
+}
+
+// 2
+function closeCompare(a, b, margin = 0) {
+    if (b - margin <= a && a <= b + margin) {
+        return 0;
+    }
+    if (a < b) {
+        return -1;
+    }
+    return 1;
+
+// 3
+    function or(a, b) {
+        return !(!a && !b);
+    }
+// true, false = true how the function works because a! is falsy and b! this would give false for !a && !b but we use !(...) so the outcome of what is inside the round brackets we invert it false is turned into truthy
+// true, true = true
+// false, true = true
+// false, false = false
+
+    function xor(a, b) {
+        return !(!a && !b ) && !(a && b); // we check of at least one of the arguments is truthy and after that && we want to ensure that both a and be are not truthy at the same time
+    }
+
+// 4
+    function _if(bool, func1, func2) {
+        if (bool) {
+            return func1();
+        }
+        return func2();
+    }
+
+// 5
+    function rentalCarCost(rentedDays) {
+        if (rentedDays >= 7) {
+            return (rentedDays * 40) - 50;
```

Comment:
Please don't use magic numbers. Magic number in the code is a number without any explanation. Sucha a number should be kept in a well named variable. 
```
const costOfCarRentalForADayInDollars = 40;
```
or something similar

Reply:
Okay, I understand the logic behind that. 
If there would've been a change needed, for example if the rate is increased, then we have to change the value in that const* and not every 'magic number'